### ISSUE TITLE
Graphics on more than one line

### ISSUE THREAD

**Issue by JamaicanUser** [nvaccess/nvda#5661]
Moving with the down and up arrows will announce 'graphic' for each line. I do not think this should be the case. Like in Jaws for Windows, I think NVDA should present the Alt Text in one line, which is sensible, since it is one graphic not two or three. This then can be applied to the Say all functionality. Providing a keystroke to toggle graphics reporting could also be explored, but this is not the aim of this ticket.

**Comment by jcsteh** [comment]
This could result in extremely long lines. For example, if the alt text of a graphic is 10000 characters long, you will get a line which is 10000 characters long. Alt text really shouldn't be used like this, but in the wild, we can't make such assumptions.

See also the discussion in #5541 and #374 . These issues discuss how these multi line links, graphics, etc. are spoken (rather than merging them into a single line), which is possibly more what you want.
